src/listener.c

```diff
@@ -454,6 +466,17 @@ init_listener(struct Listener *listener, const struct Table_head *tables, struct
         return result;
     }
 
+#ifdef SO_REUSEPORT
```

Comment:
I would prefer if we could ifdef around the smallest possible section of code (like `initiate_server_connect()`):
```
#ifdef SO_REUSEPORT
	result = setsockopt(sockfd, SOL_SOCKET, SO_REUSEPORT, &on, sizeof(on));
#else
        result = -ENOSYS;
#endif
```